network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -86,6 +86,40 @@ <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
     <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
   </div>
 </div>
+
+</div> <!-- end .container -->
```

Comment:
given that we don't have this anywhere else, let's not add start/end marker comments - if this is something we want to do, let's have a separate issue for starting that work, and then at the very least let's do this as template comments, not HTML comments, so that it doesn't show up on the page we serve.